interpreter/binary/decode.ml

```diff
@@ -216,6 +218,16 @@ let math_prefix s =
   | 0x07l -> i64_trunc_sat_f64_u
   | n -> illegal s pos (I32.to_int_u n)
 
+let simd_prefix s =
+  let pos = pos s in
+  match vu32 s with
+  | 0x0cl -> v128_const (at v128 s)
```

Comment:
missing load/store

Reply:
If you don't mind, those will come in a future PR, this commit is sufficient to run simd_i32x4_arith.wast, which does not have load/stores.

Reply:
Yep, that's OK. I was just surprised to see them below in the encoder, but not here.

Reply:
Oh yes, I probably messed up `git add -p` and `rebase` somewhere in the flow, thanks!